frontend/src/citizen-frontend/calendar/WeekElem.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+import colors from 'lib-customizations/common'
+import { DailyReservationData } from './api'
+import DayElem from './DayElem'
+import { useTranslation } from '../localization'
+
+export interface WeekProps {
+  weekNumber: number
+  data: DailyReservationData[]
+}
+
+export default React.memo(function WeekElem({ weekNumber, data }: WeekProps) {
+  const i18n = useTranslation()
+  return (
+    <div>
+      <WeekDiv>
+        {i18n.common.datetime} {weekNumber}
```

Comment:
Shouldn't this be `i18n.common.datetime.weekShort`?

Reply:
whoops, indeed